source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,114 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Major Release](https://docs.mattermost.com/upgrade/release-definitions.html#major-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - **IMPORTANT:** Session length configuration settings have changed from days to hours. If anything goes awry during a server upgrade, it could result in users not being able to log in. For customers who use environment variables for these config settings, the automatic migration won't work, and new configuration  environment variables must be added for: ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=``, ``MM_SERVICESETTINGS_SESSIONLENGTHMOBILEINHOURS=``, and ``MM_SERVICESETTINGS_SESSIONLENGTHSSOINHOURS=``. The values need to be 24x the existing values in days. For example, if ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINDAYS=30``, they should set ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=720``.
```

Comment:
```suggestion
 - **IMPORTANT:** Session length configuration settings have changed from using a unit of _days_ to _hours_. Instances using a config.json file or a database configuration for the following values should be automatically migrated to the new units, but instances using environment variables must make the following changes:
 1. replace `MM_SERVICESETTINGS_SESSIONLENGTHWEBINDAYS` with `MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS` (x24 the value).
 2. replace `MM_SERVICESETTINGS_SESSIONLENGTHMOBILEINDAYS` with `MM_SERVICESETTINGS_SESSIONLENGTHMOBILEINHOURS` (x24 the value).
 3. replace `MM_SERVICESETTINGS_SESSIONLENGTHSSOINDAYS` with `MM_SERVICESETTINGS_SESSIONLENGTHSSOINHOURS` (x24 the value).

Reply:
@cwarnermm @jasonblais Let me know if you think this is clearer and if you have any suggested edits.

Reply:
@mkraft I like the clarity you've introduced with this update. Thank you!

Reply:
Thank you! This sounds great. One clarification - what happens when someone has used environment variables, will it reset to default as mentioned by Jesse above?

Reply:
Yes, [what Jesse wrote](https://github.com/mattermost/docs/pull/5677#issuecomment-1147803623) is correct. If an instance using environment variables forgets to add the replacement `MM_SERVICESETTINGS_SESSIONLENGTH*INHOURS` setting(s) the instance will revert to using the default value in either the database or the config.json file.

Reply:
Still not 100% on this. My questions are:

If a customer/admin **has not** set these, then its all "in-product" defaults and there is no action needed. Correct?

Then if they are used:

A. If a customer/admin **has** set these, but does not change them prior to upgrade, then we ignore the custom settings (of days) and just apply the "in-product" hour defaults? Is this the case?

or

B. If a customer/admin **has** set these, but does not change them prior to upgrade, then we translate the custom settings to the new one and x24 (go from days to hours). Is this the case?